src/api/receipt.ts

```
@@ -0,0 +1,33 @@
+  tableId: number
+): Promise<{ id: string }> => {
+  try {
+    const response = await axios.post<{ id: string }>(
+      `/api/v1/receipts?storeId=${storeId}&tableId=${tableId}`
+    );
+    console.log('영수증 생성 성공:', response.data);
+    return response.data;
+  } catch (error) {
+    throw error;
+  }
+};
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)